src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Main {
+    public static final int NUMBER_OF_FIGURES = 6;
+    public static final int HALF_OF_THE_FIGURES = NUMBER_OF_FIGURES / 2;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[NUMBER_OF_FIGURES];
+
+        for (int i = 0; i < NUMBER_OF_FIGURES; i++) {
+            if (i < HALF_OF_THE_FIGURES) {
+                figures[i] = figureSupplier.getRandomFigure();
+                continue;
+            }
+            figures[i] = figureSupplier.getDefaultFigure();
```

Comment:
Can we write a code without `if` in every iteration of a loop?

Reply:
I don't understand exactly why i can't use 'if' in a loop?